keyboards/coseyfannitutti/mullet/readme.md

```diff
@@ -0,0 +1,15 @@
+# mullet
+
+![mullet](imgur.com image replace me!)
```

Comment:
```suggestion
![mullet](https://i.imgur.com/EBOMbhH.jpg)
```

Reply:
Could you supply an image here, or remove the line?